gabarit/template_num/num_project/package_name/models_training/model_keras.py

```diff
@@ -219,13 +257,18 @@ def fit(self, x_train, y_train, x_valid=None, y_valid=None, with_shuffle: bool =
 
         # Also get y_valid as numpy & get validation_data (tuple) if available
         validation_data: Optional[tuple] = None  # Def. None if y_valid is None
-        if y_valid is not None:
+        if x_valid is not None and y_valid is not None:
```

Comment:
Why has this line changed? It has not been changed in ModelKeras for the NLP template